sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_retrieval/_retrieval.py

```diff
@@ -110,7 +110,7 @@ class RetrievalEvaluator:
 
     .. code-block:: python
 
-        chat_eval = RetrievalScoreEvaluator(model_config)
+        chat_eval = RetrievalEvaluator(model_config)
```

Comment:
For the conversation payload below, i thought the `context` field has been changed a string, right?

Reply:
Good point, I'll change the doc to reflect that.